packages/calcite-components/src/utils/resources.ts

```diff
@@ -41,3 +41,10 @@ export const CSS_UTILITY = {
 export const TEXT = {
   loading: "Loading",
 };
+
+export const debounceValues = {
```

Comment:
For consistency, this should use `CONSTANT_CASE`. `Values` is implied, so WDYT about `DEBOUNCE`?